scripts/dynamic_prompting.py

```diff
@@ -196,6 +180,13 @@ def ui(self, is_img2img):
                             elem_id="magic-prompt-model",
                         )
 
+                        magic_blocklist_regex = gr.Textbox(
+                            label="Magic prompt blocklist regex",
+                            value="",
+                            elem_id="magic-prompt-blocklist-regex",
+                            placeholder="Regular expression for blocking terms out of the generated prompt",
```

Comment:
Could you add an example regex so that less technical users can get a sense of how to use it?